aiven/client/cli.py

```diff
@@ -2952,10 +2963,23 @@ def service__topic_update(self) -> None:
             retention_bytes=self.args.retention_bytes,
             retention_hours=self.args.retention,
             retention_ms=self.args.retention_ms,
+            remote_storage_enable=self._remote_storage_enable(),
+            local_retention_ms=self.args.local_retention_ms,
+            local_retention_bytes=self.args.local_retention_bytes,
             tags=tags,
         )
         print(response["message"])
 
+    def _remote_storage_enable(self) -> bool | None:
+        if self.args.remote_storage_enable and self.args.remote_storage_disable:
+            raise argx.UserError("Only set at most one of --remote-storage-enable and --remote-storage-disable")
```

Comment:
Schrödinger remote storage mode on 😄

Reply:
In this case probably isn't worth but we could always add a param to the annotation to do something like the following in case of N mutually exclusive options:
```
group = parser.add_mutually_exclusive_group()
group.add_argument('-a', action='store_true')
group.add_argument('-b', action='store_true')
```